src/sonar_view/SonarView.h

```diff
@@ -20,12 +20,12 @@ class QDESIGNER_WIDGET_EXPORT SonarView : public ImageViewOld
 
 public slots:    
 	void setOpenGL(bool flag);
-        void setSonarScan(const char *data, int size, double angle,double timeBetweenBins ,bool fromBearing=false);
+    void setSonarScan(const char *data, int size, double angle,double timeBetweenBins ,bool fromBearing=false);
```

Comment:
Please do not reformat code without a change

If you unhappy (or the indentation is wrong in general) please create a separate commit out of this